src/ducks/SANCharts/ChartPage.module.scss

```diff
@@ -24,7 +24,7 @@
 }
 
 .metrics {
-  padding: 10px 20px;
+  position: relative;
```

Comment:
Apply necessary `className` to the element, instead of breaking the ui flow in places where this component is used